doc/TechnicalOverview.md

```diff
@@ -77,8 +77,8 @@ a fully qualified URL.
 Monsteer extends the spike reader and writer interfaces defined in the Brion plugin
 API. When a library is linked against the Monsteer library, those plugins are
 automatically instantiated. Here Monsteer library provides schema starting with
-"Monsteer" keyword, such as "monsteer://" for spike streaming and "MONSTEER_steer://" for
-steering.
+"Monsteer" keyword, such as "monsteer://" for spike streaming and
+"nestSimulator://" for steering.
```

Comment:
Sorry, I'm not sure what answer you are looking for here. It has nothing to do with Brion, that's correct. It is a monsteer::Simulator class, which uses ZEQ internally. To use it, people need to provide a valid scheme to the constructor so that it can instantiate a plugin (in this, "nest://").